contributors/devel/kubemark-guide.md

```diff
@@ -202,9 +234,18 @@ Because there's no easy way of mocking other managers (e.g. VolumeManager), they
 are not supported in Kubemark (e.g. we can't schedule Pods with volumes in them
 yet).
 
-As the time passes more fakes will probably be plugged into HollowNodes, but
-it's crucial to make it as simple as possible to allow running a big number of
-Hollows on a single core.
+We currently plan to extend kubemark along the following directions:
+- As you would have noticed at places above, we aim to make kubemark more structured
+  and easy to run across various providers without having to tweak the setup scripts,
+  using a well-defined kubemark-provider API.
+- Allow kubemark to run on various distros (GCI, debian, redhat, etc) for any
+  given provider.
+- Make Kubemark performance on ci-tests mimic real cluster ci-tests on various metrics
```

Comment:
@xiang90 Good suggestion. Added it to the point.

@hongchaodeng Changed the sentence to illustrate better what we possibly mean by various metrics. For reference, we use the CPU and memory usage as the metrics to compare two test runs in [kubernetes/contrib/compare](https://github.com/kubernetes/contrib/tree/master/compare). However, further discussion is required to come up with the desired set of metrics that could ascertain
similarity b/w perf of kubemark and real cluster.
(Comments? @wojtek-t @gmarek )